src/plugins/equalheight/reports/1-acr-wcag21.json

```
@@ -0,0 +1,494 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json-ld",
+
+  "@type": "acr:ConformanceReport",
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/wet-boew/demos/equalheight/equalheight-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Only on the enhanced and basic HTML version of the add to euqal height component.",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ".wb-euqalheight"
+    }
+  },
+
+  "earl:assertedBy": {
+    "@id": "_:acrAssertor",
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Hongbin Yu (Github: @hongbinyu413)",
+      "foaf:homepage": "https://github.com/hongbinyu413",
+      "@type": ["earl:Assertor", "foaf:Person"]
+    }
+  },
+
+  "dct:created": "2022-09-24",
+  "dct:issued": "2023-03-02",
+  "dct:modified": "2023-06-19",
+  "dct:description": "Add to euqal height plugin conformance report at WCAG 2.1 Level AA. That is focussed on the add euqal height used in a context as demoed by the working example (Test subject).",
+
+  "acr:standard": "https://www.w3.org/TR/WCAG21",
+  "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+
+  "acr:conformance": "acr:satisfied",
+  "acr:aggregatedScore": "100%",
+
+  "assessments": {
+    "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json": {
+      "dct:title": "Add to euqal height WCAG 2.1 Level AA assessment",
+      "dct:references": "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21-en.html",
+      "dct:language": "en",
+      "dct:date": "2023-05-02",
+      "@type": "acr:AssessmentReport"
+    }
+  },
+
+  "acr:audit": [
+    {
+      "earl:assertedBy": {
+        "foaf:name": "Pierre Dubois (Github: @duboisp)",
+        "foaf:homepage": "https://github.com/duboisp",
+        "@type": ["earl:Assertor", "foaf:Person"]
+      },
+      "@type": ["acr:AuditReport", "earl:Assertion"],
+      "earl:mode": "earl:manual",
+      "acr:accuracy": "acr:accurate",
+      "dct:description": "Confirm the score of 100%. There is no work items and the WCAG 2.2 numbering, as today, has changed since but I didn't modified them.",
+      "dct:issued": "2023-03-02",
+      "acr:assessment": "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json",
+      "acr:auditNote": [ ]
+    }
+  ],
+
+  "wf:task": [ ],
+
+  "dct:source": "act:standard/sets/wcag2x/wcag21_all_levelAA.json",
+  "acr:conformity": [
+    {
+      "acr:requirement": "WCAG21:non-text-content",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json"
```

This assessment link, and the subsequent, are not correct.

Here the final URL once this PR get merged
```suggestion
        "https://wet-boew.github.io/wet-boew/demos/equalheight/reports/1-assessment-wcag21.json"
```